wwt_api_client/constellations/handles.py

```diff
@@ -532,6 +533,7 @@ def add_scene_from_place(self, place: Place) -> str:
             content=content,
             text=text,
             outgoing_url=outgoing_url,
+            publish=True,
```

Comment:
Users of this API might want to influence the publication status used here, so I think it would make sense to add a `publish=True` defaulted keyword argument to the function.